src/window.rs

```diff
@@ -62,6 +67,9 @@ impl fmt::Debug for Window {
     }
 }
 
+/// This will close the [`Window`].
+///
+/// See [`Window`] for more details.
```

Comment:
I think the general trend is to put documentation on the `drop()` method instead of the trait implementation.

Reply:
Uh, true, but you aren't supposed to actually call `Drop::drop()`? Looking at `std` I couldn't find any `Drop` implementation documented.

Will go ahead with moving the documentation to the method.